### PR TITLE
Add scorer purpose

### PR DESCRIPTION
**Pull request by ankrgyl**
This allows us to exclude LLM-as-a-judge calls from certain metrics (e.g. token counts)

### CODE REVIEW

File: js/oai.ts [hunk 1]
```diff
@@ -69,6 +72,7 @@ export function buildOpenAIClient(options: OpenAIAuth): OpenAI {
 }
 
 declare global {
+  // eslint-disable-next-line
```
Comment (manugoyal) [inline js/oai.ts:75]: is there a more specific warning we can disable? or maybe a comment to explain what we are ignoring?

File: py/autoevals/oai.py [hunk 2]
```diff
@@ -18,7 +18,12 @@ class OpenAIWrapper:
     RateLimitError: Exception
 
 
+_WRAPPED_OPENAI = False
```
Comment (manugoyal) [inline py/autoevals/oai.py:21]: Can we avoid the global variable by returning this boolean as part of `prepare_openai`?

**Review by manugoyal — APPROVED** [review]

File: js/oai.ts (hunk 1 above)
Comment (ankrgyl) [reply js/oai.ts:75]: Yep added more specific one

File: py/autoevals/oai.py (hunk 2 above)
Comment (ankrgyl) [reply py/autoevals/oai.py]: good call

**Pull request merged**